app/components/ui/QrCodeModal/QrCodeModal.module.scss

```diff
@@ -34,8 +34,6 @@
   position: absolute;
   right: 30px;
   top: 30px;
-  height: 12px;
-  width: 12px;
```

Comment:
I removed sizing of the close icon on the QR Code Modal to align with the sizing on existing modals in our code base. In discussions with Dongyeon for this ClinicianAction modal, we decided to use the existing "close" icons for our modals rather than his new designs. Thus I changed it here as well for consistency.